src/view/actions/sendEvent.jsx

```diff
@@ -121,7 +121,7 @@ const SendEvent = () => {
           <ExtensionViewForm
             initialValues={getInitialValues({ initInfo })}
             getSettings={getSettings}
-            validationSchema={validationSchema}
+            formikStateValidationSchema={validationSchema}
```

Comment:
As you'll see, we now have `validateFormikState`, `formikStateValidationSchema`, and `validateNonFormikState` options.